database/migrations/2023_10_19_010000_change_is_limited_to_cart_item_to_apply_coupon_on_enum.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Igniter\Coupons\Database\Migrations;
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeIsLimitedToCartItemToApplyCouponOnEnum extends Migration
+{
+    public function up()
+    {
+        Schema::table('igniter_coupons', function (Blueprint $table) {
```

Comment:
Lets split this so we can copy existing is_limited_to_cart_item values over to the new enum column before actually dropping the column. This way users who already have coupons with is_limited_to_cart_item enabled won't have that changed to whole_cart.